india_compliance/gst_india/doctype/purchase_reconciliation_tool/purchase_reconciliation_tool.js

```diff
@@ -143,9 +149,14 @@ frappe.ui.form.on("Purchase Reconciliation Tool", {
 
     purchase_period(frm) {
         fetch_date_range(frm, "purchase");
+        let description = frm.doc.purchase_period == "Last Fiscal Year" ? fiscal_start +' to '+ fiscal_end : ""
+        frm.get_field("purchase_period").set_description(description);
```

Comment:
This should be for all periods. Date range is already fetched. We can use directly that to show in description.